Декоративные элементы всех страниц

deniskirshenin opened the pull request:
добавил декоративные элементы
добавил состояния элементов
добавил отступы и размеры

---
:mortar_board: [Декоративные элементы всех страниц](https://up.htmlacademy.ru/htmlcss/35/user/2193861/tasks/13)

:boom: https://htmlacademy-htmlcss.github.io/2193861-sedona-35/7/

deniskirshenin committed:
Декоративные элементы всех страниц

добавил декоративные элементы
добавил состояния элементов
добавил отступы и размеры

smart-grizzly commented on styles/styles.css:
```diff
@@ -409,31 +633,97 @@ body {
+  border: none;
+  width: 452px;
+  padding-top: 14px;
+  padding-right: auto;
```
Нет такого свойства у `padding`, `auto` - авто есть только в маргина

smart-grizzly commented on catalog.html:
```diff
@@ -10,7 +10,7 @@
 <body>
   <header class="main-header" data-test="header">
```
https://validator.w3.org/nu/#textarea
Ошибки критичные в валидаторе!

deniskirshenin committed:
Отправляю на финальную проверку

Исправил ошибки в валидаторе
Поправил верстку для инлайн SVG
Изменил версту и стили для class="hotel-search-link"

deniskirshenin committed:
Merge branch 'htmlacademy-htmlcss:master' into master
